Include/internal/pycore_optimizer.h

```diff
@@ -33,6 +33,7 @@ struct _Py_UopsSymbol {
     int flags;  // 0 bits: Top; 2 or more bits: Bottom
     PyTypeObject *typ;  // Borrowed reference
     PyObject *const_val;  // Owned reference (!)
+    int32_t typ_version; // currently stores type version
```

Comment:
Minor: can you change this to `type_version` everywhere?

```suggestion
    int32_t type_version;
```

Reply:
Also, should this be an unsigned int, since that's what `tp_version_tag` is?